src/handlers/motions/helpers.ts

```diff
@@ -9,17 +9,34 @@ export const getMotionSide = (vote: BigNumber): MotionSide =>
   vote.eq(MotionVote.YAY) ? MotionSide.YAY : MotionSide.NAY;
 
 export const updateMotionInDB = async (
-  id: string,
-  motionData: MotionData,
+  motionData: ColonyMotion,
   showInActionsList?: boolean,
 ): Promise<void> => {
-  await mutate('updateColonyAction', {
+  await mutate('updateColonyMotion', {
```

Comment:
Not sure this is possible now that the motion data is an Amplify model, it will only accept the motion ID:
<img width="324" alt="image" src="https://github.com/JoinColony/block-ingestor/assets/112586815/41dbfe13-f636-45c7-a31e-2d8ab14fffbf">

Reply:
If that's the case, do you think we could create a utility function that peforms both mutations @ArmandoGraterol? To simulate the previous behaviour.

Reply:
Sure 👍 I'll take a look

Reply:
@willm30 I've created a helper function that contains both mutations, let me know if something like that is what you had in mind.